examples/seismic/abc_methods/02_damping.ipynb

```diff
@@ -0,0 +1,1257 @@
+    "\\end{equation}\n",
+    "\n",
+    "- In the white region:\n",
+    "\n",
+    "\\begin{equation}\n",
+    "u_{tt}(x,z,t)-c^2(x,z)^\\Delta(u(x,z,t))=c^2(x,z)f(x,z,t),\n",
+    "\\end{equation}\n",
+    "\n",
+    "For this reason, we use the structure of the *subdomains* to represent the white region and the blue region.\n",
+    "\n",
+    "**Observation:** Note that we can describe the blue region in different ways, that is, the way we choose here is not the only possible discretization for that region.\n",
+    "\n",
+    "First, we define the white region, naming this region as *d0*, which is defined by the following pairs of points $(x,z)$:\n",
+    "\n",
+    "- $x\\in\\{npmlx,nptx-npmlx\\}$ and $z\\in\\{0,nptz-npmlz\\}$.\n",
+    "\n",
+    "In the language of *subdomains* *d0  it is written as:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class d0domain(SubDomain):\n",
+    "    name = 'd0'\n",
+    "    def define(self, dimensions):\n",
+    "        x, z = dimensions\n",
+    "        return {x: ('middle', npmlx, npmlx), z: ('middle', 0, npmlz)}\n",
+    "d0_domain = d0domain()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The blue region will be the union of the following regions:\n",
+    "\n",
+    "- *d1* represents the left range in the direction *x*, where the pairs $(x,z)$ satisfy: $x\\in\\{0,npmlx\\}$ and $z\\in\\{0,nptz\\}$;\n",
+    "- *d2* represents the rigth range in the direction *x*, where the pairs $(x,z)$ satisfy: $x\\in\\{nptx-npmlx,nptx\\}$ and $z\\in\\{0,nptz\\}$;\n",
+    "- *d3* represents the left range in the direction *y*, where the pairs $(x,z)$ satisfy: $x\\in\\{npmlx,nptx-npmlx\\}$ and $z\\in\\{nptz-npmlz,nptz\\}$;\n",
+    "\n",
+    "Thus, the regions *d1*, *d2* and *d3* are described as follows in the language of *subdomains*:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class d1domain(SubDomain):\n",
+    "    name = 'd1'\n",
+    "    def define(self, dimensions):\n",
+    "        x, z = dimensions\n",
+    "        return {x: ('left',npmlx), z: z}\n",
+    "d1_domain = d1domain()\n",
+    "\n",
+    "class d2domain(SubDomain):\n",
+    "    name = 'd2'\n",
+    "    def define(self, dimensions):\n",
+    "        x, z = dimensions\n",
+    "        return {x: ('right',npmlx), z: z}\n",
+    "d2_domain = d2domain()\n",
+    "\n",
+    "class d3domain(SubDomain):\n",
+    "    name = 'd3'\n",
+    "    def define(self, dimensions):\n",
+    "        x, z = dimensions\n",
+    "        return {x: ('middle', npmlx, npmlx), z: ('right',npmlz)}\n",
+    "d3_domain = d3domain()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The figure below represents the division of domains that we did previously:\n",
```

Comment:
Different colours for each subdomain in the plot?

Reply:
The purpose of this domain representation is to indicate the absorption region, where the damping functions will act to reduce reflections.